Parsers/ASimUserManagement/Parsers/vimUserManagementSentinelOne.yaml

```diff
@@ -0,0 +1,185 @@
+      3715, "PasswordReset", "User Reset Password by Admin Request", "",
+      5006, "GroupDeleted", "Group Deleted", "",
+      5008, "GroupCreated", "User created a Manual or Pinned Group", "",
+      5011, "GroupModified", "Group Policy Reverted", "Newpolicy",
+  ];
+  let parser = (
+    starttime:datetime=datetime(null), 
+    endtime:datetime=datetime(null),
+    targetusername_has:string="*", 
+    actorusername_has:string="",
+    targetdomain_has:string="*",
+    anydomain_has:string="*",
+    disabled:bool=false
+    ) {
+      SentinelOne_CL
+      | where not(disabled)
+          and event_name_s == "Activities."
+          and (isnull(starttime) or TimeGenerated >= starttime) and (isnull(endtime) or TimeGenerated <= endtime)
+          and activityType_d in (23, 24, 25, 37, 102, 110, 111, 140, 141, 142, 3522, 3523, 3524, 3710, 3711, 3715, 5006, 5008, 5011)
+          and (targetusername_has == "*" or DataFields_s has targetusername_has)
+          and (actorusername_has == "*" or DataFields_s has actorusername_has)
+          and targetdomain_has == "*"
+          and anydomain_has == "*"
+      | parse-kv DataFields_s as (byUser: string, username: string, email: string, ipAddress: string, group: string, groupName: string, name: string, oldDescription: string, oldRole: string, description: string, role: string, userScope: string, scopeLevelName: string, scopeName: string, roleName: string, modifiedFields: string, deactivationPeriodInDays: string, descriptionChanged: string, groupType: string) with (pair_delimiter=",", kv_delimiter=":", quote='"')
+      | parse modifiedFields with 'Modified fields: ' ModifiedFields:string
+      | parse description_s with * "with id=" id: string "," restOfMessage
+      | lookup EventTypeLookup on activityType_d
+      | extend 
+          ActorUsername = iff(activityType_d == 102, "SentinelOne", coalesce(byUser, username, email)),        
+          GroupName = coalesce(group, groupName, name),
+          TargetUsername = iff(isnotempty(byUser), username, ""),
+          PreviousPropertyValue = coalesce(oldDescription, oldRole),
+          NewPropertyValue = coalesce(description, role)
+      | where (targetusername_has == "*" or TargetUsername has targetusername_has)
+          and (actorusername_has == "*" or ActorUsername has actorusername_has)
+      | extend GroupName = iff(GroupName == "null", "", GroupName)
+      | project-rename
+          EventStartTime = createdAt_t,
+          SrcIpAddr = ipAddress,
+          EventUid = _ItemId,
+          ActorUserId = id,
+          GroupId = groupId_s,
+          EventMessage = primaryDescription_s,
+          EventOriginalUid = activityUuid_g
+      | extend
+          EventCount = int(1),
+          EventResult = "Success",
+          DvcAction = "Allowed",
+          EventSeverity = "Informational",
+          EventSchema = "UserManagement",
+          EventSchemaVersion = "0.1.1",
+          EventProduct = "SentinelOne",
```

Comment:
Please add SentinelOne as supported value for EventVendor and EventProduct

Reply:
Done